lib/services/status-bar/status-bar.js

```diff
@@ -3,23 +3,25 @@
 import React from "react";
 import { observer } from "mobx-react";
 
-import type Kernel from "./../kernel";
-import typeof store from "./../store";
+import type Kernel from "../../kernel";
+import type { Store } from "../../store";
 
 type Props = {
-  store: { kernel: ?Kernel, configMapping: Map<string, mixed> },
+  store: Store,
   onClick: Function
 };
 
-const StatusBar = observer(
-  ({ store: { kernel, configMapping }, onClick }: Props) => {
+@observer
```

Comment:
Is this change prep for #1284?

Reply:
not that PR but the outstanding one #1631

Reply:
We may not need to change the status-bar component into class one, and in the future, react components will be preferred to be functional component.
Thus please hold on it a bit.